packages/components/src/duotone-picker/duotone-picker.js

```diff
@@ -35,10 +35,24 @@ function DuotonePicker( {
 	return (
 		<CircularOptionPicker
 			options={ duotonePalette.map( ( { colors, slug, name } ) => {
-				const style = {
-					background: getGradientFromCSSColors( colors, '135deg' ),
-					color: 'transparent',
-				};
+				const iconProps =
+					slug === 'unset'
+						? {
+								style: {
+									background:
+										'#fff linear-gradient(-45deg, transparent 48%, #ddd 48%, #ddd 52%, transparent 52%)',
+									boxShadow: 'none',
```

Comment:
I tried here to use `boxShadow: 'none !important',`, but that didn't work. Essentially, if we can make this box shadow be !important, we don't have to change the CircularOptionPicker component directly.

Alternatively can we migrate to using `ColorIndicator` instead?